HANDBOOK.md

```diff
@@ -207,13 +233,202 @@ import { ManifestResolver } from '@salesforce/source-deploy-retrieve';
+|      ├── a.field-meta.xml
+|      ├── b.field-meta.xml
+|      ├── c.field-meta.xml
+
+```
+
+### The `digitalExperienceAdapter`:
+
+Source Adapter for DigitalExperience metadata types. This metadata type is a bundled type of the format.
+
+🧽 This is an example of SDR extending one of the "classes of metadata" to add support for DigitalExperienceBundle.
+Ideally this adapter would've been named something without the specific metadata type in its name, but until another type comes along and uses this adapter it'll be ok.
+
+**Example Structure**:
+
+```text
+site/
+├── foos/
+|   ├── sfdc_cms__appPage/
+|   |   ├── mainAppPage/
+|   |   |  ├── _meta.json
+|   |   |  ├── content.json
+|   ├── sfdc_cms__view/
+|   |   ├── view1/
+|   |   |  ├── _meta.json
+|   |   |  ├── content.json
+|   |   |  ├── fr.json
+|   |   |  ├── en.json
+|   |   ├── view2/
+|   |   |  ├── _meta.json
+|   |   |  ├── content.json
+|   |   |  ├── ar.json
+|   ├── foos.digitalExperience-meta.xml
+content/
+├── bars/
+|   ├── bars.digitalExperience-meta.xml
+```
+
+In the above structure the metadata xml file ending with "digitalExperience-meta.xml" belongs to DigitalExperienceBundle MD type.
+The "\_meta.json" files are child metadata files of DigitalExperienceBundle belonging to DigitalExperience MD type. The rest of the files in the
+corresponding folder are the contents to the DigitalExperience metadata. So, in case of DigitalExperience the metadata file is a JSON file
+and not an XML file.
+
+### The `matchingContentAdapter`:
+
+This adapter is used for `ApexClass` or other types where there is a "content" file and an .xml file. In source-format, an `ApexClass` is made up of two files, the `.cls` which contains the actual code from the class, and an accompanying "meta" file
+
+```text
+classes/
+ ├── myApexClass.cls
+ ├── myApexClass.cls-meta.xml
+```
+
+### The `mixedContentAdapter`:
+
+Handles types with mixed content. Mixed content means there are one or more additional file(s) associated with a component with any file extension. Even an entire folder can be considered "the content".
+
+**Example Types**:
+
+StaticResources, Documents, Bundle Types
+
+**Example Structures**:
+
+```text
+staticresources/
+├── data.resource
+├── data.resource-meta.xml/
+
+bars/
+├── myBar.xyz
+├── myBar.ext2-meta.xml
+```
+
+### Transformers
+
+So now that we're able to recognize different "classes" of metadata based on either their file structure, their extensions, matching files, or any of the adapters listed above, we can start to convert them from source, or metadata format, to the other
+SDR accomplishes this conversion with "Transformers" which will transform the metadata. Similar to how there was a default adapter that handled most of the metadata types, this is true for transformers as well, but because we've committed to doing all of this work on the client, we have to support some complicated types, and have a few more transformers to help handle the edge-case types.
```

Comment:
Add comma `SDR accomplishes this conversion with "Transformers", which` or change to `SDR accomplishes this conversion with "Transformers" that`